### PR TITLE
Update version-5.3.md

### PR DESCRIPTION
**Pull request by letzya**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

___

## **Type**
documentation, enhancement

___

## **Description**
- Clarified that Tyk OAS APIs are now out of Early Access, ensuring backward compatibility for future versions, but not with versions prior to 5.3.0.
- Updated the documentation to reflect that Tyk OAS API definitions will automatically be updated to the latest format by the Tyk Dashboard during the upgrade process.
- Highlighted the non-reversible nature of upgrading Tyk OAS API definitions and the need for backups when downgrading.
- Provided updated guidance for MDCB users on the compatibility of Tyk OAS APIs with MDCB versions, advising waiting for MDCB v2.5 before upgrading.

___

## **Changes walkthrough**
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Clarification and Updates on Tyk OAS APIs Backward Compatibility and </code><br><code>MDCB Guidance</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md
<li>Clarified the backward compatibility status of Tyk OAS APIs <br>post-upgrade.<br> <li> Highlighted the automatic update of Tyk OAS API definitions to the <br>latest format by Tyk Dashboard upon upgrade.<br> <li> Emphasized the non-reversible nature of the upgrade regarding Tyk OAS <br>API definitions.<br> <li> Updated guidance for MDCB users regarding the compatibility of Tyk OAS <br>APIs with MDCB versions.

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4348/files#diff-14b1be0b2206c9ee21cb1002b55361919932e84558f614c4342f05b8eb5431f8">+19/-28</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> ✨ **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 1]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
-
-- Historic Tyk OAS API Definition Compatibility: 
-  - Tyk OAS API Definitions prior to version 5.3.0 are not compatible with Tyk Gateway v5.3.X.
+#### Tyk OAS APIs are not backwards-compatible
+
+This upgrade transitions Tyk OAS APIs out of [Early Access]({{< ref "frequently-asked-questions/using-early-access-features" >}}). 
+
+- **Out of Early access**
+  - This means that from now on, all Tyk OAS APIs will be backwards compatible and in case of a downgrade from 5.3.X to 5.3.0, the Tyk OAS API definitions will always work.
+- **Not Backwards Compatible and Non-Reversible Change**
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]: Could we use heading of **Backwards Compatibility**. The list items below mention that it is not backwards compatible and not reversible.

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 2]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
+  - Not [backwards compatible](https://en.wikipedia.org/wiki/Backward_compatibility#:~:text=Backward%20compatibility%20(sometimes%20known%20as,especially%20in%20telecommunications%20and%20computing.) means that the new Tyk OAS API format created by Tyk Gateway/Dashboard v5.3.X does not work with older versions of Tyk Gateway/Dashboard, i.e. you cannot export these API definitions from a v5.3.X Tyk Dashboard and import to an earlier version.
+  - The upgrade is not reversible i.e. you cannot use version 5.3.X Tyk OAS API definitions with an older Tyk Dashboard.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
  - The upgrade is not reversible, i.e. you cannot use version 5.3.X Tyk OAS API definitions with an older version of Tyk Dashboard.
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 3]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
+  - This means that in case you want or need to downgrade or revert to your previous version of Tyk, you will need to restore these API definitions from a backup. Please consult our guidance documentation for how to [backup Tyk APIs and Policies]({{ < ref "" >}})
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
  - This means that if you wish to downgrade or revert to your previous version of Tyk, you will need to restore these API definitions from a backup. Please consult our guidance documentation for how to [backup Tyk APIs and Policies]({{ < ref "" >}})
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 4]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
+  - If you are not using Tyk OAS APIs Tyk will keep its backward compatibility standards.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
  - If you are not using Tyk OAS APIs, Tyk will maintain backward compatibility standards.
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 5]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
+- **Historic Tyk OAS API Definition Compatibility**
+  - Tyk OAS API Definitions prior to v5.3.0 are not compatible with Tyk Gateway v5.3.X.
   - This means that any Tyk OAS APIs created in any previous release (4.1.0-5.2.x) cannot work with the new Tyk Dashboard without being migrated to the new format.
-- After upgrade:
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md:45]: Is this a heading, e.g. **After Upgrade**

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 6]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
-<!-- #### Changed error log messages
-Important for users who monitor Tyk components using the application logs (i.e. Tyk Gateway log, Tyk Dashboard log etc.).
-We try to avoid making changes to our log messages, especially at error and critical levels. However, sometimes it's necessary. Please find the list of changes made to the application log in this release: -->
-
-<!-- The following "|Planned Breaking Changes" section is optional!
-Announce future scheduled breaking changes, e.g. Go version updates, DB driver updates etc. -->
-<!-- #### Planned Breaking Changes -->
-
+- **MDCB deployment and Tyk OAS APIs**
+  - MDCB users utilising Tyk OAS APIs **should wait for the release of MDCB v2.5** before upgrading Tyk Gateway and Dashboard to v5.3.0. This means Tyk OAS APIs will not be loaded with MDCB v2.4 or older.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md:56]: Does this mean that the backward compatibility of 2.4.x in compatibility matrix table of release notes is incorrect?

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 7]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
+  - MDCB users using Tyk OAS APIs, Tyk Dashboard 5.3.0 will only work with Tyk Gateway v5.3.0 and MDCB v2.5.X. Older versions of Tyk Gateway may not function correctly with v-5.3.0-compatible Tyk OAS API definitions.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
  - For MDCB users using Tyk OAS APIs, Tyk Dashboard 5.3.0 will only work with Tyk Gateway v5.3.0 and MDCB v2.5.X. Older versions of Tyk Gateway may not function correctly with v5.3.0 Tyk OAS API definitions.
```

**Review by dcs3spp — COMMENTED** [review]
Thanks @letzya I think the subsection headings that you have added make the breaking change section of backward compatibility clearer. I have left minor comments/suggestions

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 8]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
-- Historic Tyk OAS API Definition Compatibility: 
-  - Tyk OAS API Definitions prior to version 5.3.0 are not compatible with Tyk Gateway v5.3.X.
+#### Tyk OAS APIs are not backwards-compatible
+
+This upgrade transitions Tyk OAS APIs out of [Early Access]({{< ref "frequently-asked-questions/using-early-access-features" >}}). 
+
+- **Out of Early access**
+  - This means that from now on, all Tyk OAS APIs will be backwards compatible and in case of a downgrade from 5.3.X to 5.3.0, the Tyk OAS API definitions will always work.
+- **Not Backwards Compatible and Non-Reversible Change**
+  - Not [backwards compatible](https://en.wikipedia.org/wiki/Backward_compatibility#:~:text=Backward%20compatibility%20(sometimes%20known%20as,especially%20in%20telecommunications%20and%20computing.) means that the new Tyk OAS API format created by Tyk Gateway/Dashboard v5.3.X does not work with older versions of Tyk Gateway/Dashboard, i.e. you cannot export these API definitions from a v5.3.X Tyk Dashboard and import to an earlier version.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
(https://en.wikipedia.org/wiki/Backward_compatibility#:~:text=Backward%20compatibility%20(sometimes%20known%20as,especially%20in%20telecommunications%20and%20computing.) means that the new Tyk OAS API format created by Tyk Gateway/Dashboard v5.3.X does not work with older versions of Tyk Gateway/Dashboard, i.e. you cannot export these API definitions from a v5.3.X Tyk Dashboard and import to an earlier version.
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 9]
```diff
@@ -35,36 +35,27 @@ This release is ready for installation on your own infrastructure.
...
-
-This upgrade transitions Tyk OAS APIs out of [Early Access]({{< ref "frequently-asked-questions/using-early-access-features" >}}).
-
-- Historic Tyk OAS API Definition Compatibility: 
-  - Tyk OAS API Definitions prior to version 5.3.0 are not compatible with Tyk Gateway v5.3.X.
+#### Tyk OAS APIs are not backwards-compatible
+
+This upgrade transitions Tyk OAS APIs out of [Early Access]({{< ref "frequently-asked-questions/using-early-access-features" >}}). 
+
+- **Out of Early access**
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md:42]:
```suggestion
- **Not [backwards compatible]**
```

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md (hunk 6 above)
Comment (letzya) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md:56]: let's comment in the main pr

**Pull request merged**